.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Replace this text, including the symbols above and below it, with descriptive text about
+the change you are proposing. Please include a reference to any issues addressed or
+resolved in that text, for example "resolves #1".
+-->
+
+<!--
+IMPORTANT: As a contributor, we would like as much help as you can offer, but we only
+expect you to complete the steps in the "PR draft checklist" below. Maintainers are
+willing and ready to help pick it up from there!
+
+Please start by opening this Pull Request as a "draft". You can do this by
+clicking the arrow on the right side of the green "Create pull request" button. Follow
+the draft checklist below to move the PR out of draft state. If you accidentally created
+the PR as a non-draft, don't worry, you can still change it to a draft using the
+"Convert to draft" button on  the right side panel under the "Reviewers" section.
+-->
+
+<details><summary>Pull Request (PR) draft checklist - click to expand</summary>
+
+- [ ] Populate a descriptive title. Ensure the title does not look like "Updated
+      README.md".
```

Comment:
Perhaps we can help the contributor find a more useful way to describe their PR, in addition to saving what to avoid?  Maybe something like:

- [ ] Populate a descriptive title, including what is particular to this PR. For example, instead of "Updated README.md", use a title such as "Added testing details to the contributor section of the README".

Reply:
91e5599